LFS-1125: Split LFS forms into semantic sections

sdumitriu opened the pull request:
To check:
- are all the fields still present?
- does the new order make sense?
- should the sections have labels, or is it OK without any demarcation of the sections?

sdumitriu committed:
LFS-1125: Split LFS forms into semantic sections

sdumitriu committed:
LFS-1125: Split LFS forms into semantic sections
Renamed 'Demographics' to 'Patient information'

sashaandjic commented:
There is a problem with radio form. I created a form, entered all required info and saved it. 
![image](https://user-images.githubusercontent.com/23243244/116480337-26b1c780-a84f-11eb-91ad-58e89a6e093d.png)

When I open it, it show only one field `Radiation therapy`:
![image](https://user-images.githubusercontent.com/23243244/116480418-4f39c180-a84f-11eb-8ea2-ec23d94c8e4d.png)

When I click on edit button; other fields appear, however `Radiation therapy` field seems to be blank.
![image](https://user-images.githubusercontent.com/23243244/116480533-7db79c80-a84f-11eb-9d8d-e5b3749cc90b.png)


Start and end date now have Height and Weight fields in between them.

sashaandjic commented:
The above issue has been addressed in https://github.com/ccmbioinfo/lfs/pull/621#pullrequestreview-648706699

sashaandjic commented:
Radio and Chemo form in view mode only show the very first question and the last one even though all fields are populated.
![image](https://user-images.githubusercontent.com/23243244/116628152-82965200-a91c-11eb-929b-befa9f1a5555.png)

sdumitriu committed:
LFS-1125: Split LFS forms into semantic sections
- Use _section suffix for section node names
- Flatten section hierarchy a bit to work around LFS-1134

sashaandjic reviewed (approved):
tested

marta- reviewed (approved):
I thought we were removing the first question in the Chemo and Radio questionnaires (Chemo/Radio? yes / no / unknown).

sdumitriu commented:
> I thought we were removing the first question in the Chemo and Radio questionnaires (Chemo/Radio? yes / no / unknown).

I don't remember the conclusion, but I raised the idea that a chemo form with No selected and comments filled in explaining why it was not done ("patient is alergic to it"). However, there is a boolean Chemotherapy question in the tumor form (as well as radio, immuno, and a couple of other therapies), to which we may add Notes. Should we do that instead?

marta- commented:
> > I thought we were removing the first question in the Chemo and Radio questionnaires (Chemo/Radio? yes / no / unknown).
> 
> I don't remember the conclusion, but I raised the idea that a chemo form with No selected and comments filled in explaining why it was not done ("patient is alergic to it"). However, there is a boolean Chemotherapy question in the tumor form (as well as radio, immuno, and a couple of other therapies), to which we may add Notes. Should we do that instead?

Leave as is.
